.openpublishing.redirection.json

```diff
@@ -171,14 +171,29 @@ Old redirects (to /archive/)
       },
       {
         "source_path": "microsoft-edge/progressive-web-apps-chromium/experimental-features/index.md",
-        "redirect_url": "/microsoft-edge/progressive-web-apps-chromium/how-to/origin-trials",
+        "redirect_url": "/microsoft-edge/origin-trials/index",
+        "redirect_document_id": false
+      },
+      {
+        "source_path": "microsoft-edge/progressive-web-apps-chromium/how-to/origin-trials.md",
+        "redirect_url": "/microsoft-edge/origin-trials/index",
         "redirect_document_id": false
       },
       {
         "source_path": "microsoft-edge/progressive-web-apps-chromium/offline.md",
         "redirect_url": "/microsoft-edge/progressive-web-apps-chromium/how-to/offline",
         "redirect_document_id": false
       },
+      {
+        "source_path": "microsoft-edge/progressive-web-apps-chromium/how-to/web-app-manifests.md",
+        "redirect_url": "https://developer.mozilla.org/docs/Web/Manifest",
```

Comment:
when you try to go to
https://learn.microsoft.com/microsoft-edge/progressive-web-apps-chromium/how-to/web-app-manifests
should you end up at
https://developer.mozilla.org/docs/Web/Manifest
or at
https://developer.mozilla.org/docs/Web/Progressive_web_apps/Guides/Making_PWAs_installable#the_web_app_manifest

Reply:
The latter.

Reply:
redirected to latter Guide-type url instead of API-type url